new-pipeline/dist.html

```diff
@@ -55,7 +55,10 @@ <h2 style="font-size: 18px; margin-bottom: 0">
                 </h2>
                 <hr></hr>
                 <figure class="col-md-12">
-                  <figcaption id="dist-caption"></figcaption>
+                <div class="caption-div">
+                  <figcaption id="dist-caption-text" class="overflow ellipsis"></figcaption>
+                  <figcaption id="dist-caption-link"></figcaption>
+                </div></br></br>
```

Comment:
I think we should not use `br` here. Instead lets use `margin-bottom` or similar.